attributes/default.rb

```diff
@@ -26,7 +26,10 @@
 default['osl-openstack']['cinder']['iscsi_ips'] = []
 default['osl-openstack']['image']['glance_vol'] = nil
 default['osl-openstack']['endpoint_hostname'] = nil
+default['osl-openstack']['network_hostname'] = nil
 default['osl-openstack']['db_hostname'] = nil
+default['osl-openstack']['bind_service'] = node['ipaddress']
+default['osl-openstack']['network_node'] = false
```

Comment:
I have a basic question.

How does the controller in an openstack environment understand that there is a different network node and it should not try to do the network stuff ? Is this the line which does that ? Because to me it sounds like this attribute says this *is* the network node :)